features/attached_commands.feature

```diff
@@ -168,22 +168,22 @@ Feature: Command behaviour when attached to an UA subscription
         And I run `ua version` as non-root
         Then I will see the following on stdout:
             """
-            20.4
+            25.0
             """
         When I run `ua version` with sudo
         Then I will see the following on stdout:
             """
-            20.4
+            25.0
```

Comment:
I don't like that we have to keep updating this as we change version numbers.  Can we update this to import uaclient.version.__VERSION__. And use that throughout?